src/content/docs/configuration/providers.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Providers
+description: Understand Providers and configure new Targets.
+sidebar:
+  label: Providers
+---
+
+Daytona abstracts the management and deployment of Workspaces into 2 related concepts:
+
+1.  __Provider__  
+    A plugin that interfaces with [Daytona Server](/reference/server), responsible for Workspace deployment and lifecycle management.
+    Examples of platforms Providers may connect with include:
+    *   Container management platforms such as Docker or Podman;
+    *   Cloud hosting providers such as AWS or DigitalOcean.
+2.  __Target__  
+    A set of configuration that governs how Daytona manages and deploys Workspaces.
+    Each Target is tied to an individual Provider.
+    Daytona allows you to configure new and existing Targets, and choose between them at Workspace creation time.
+
+Daytona's architecture decentralizes the role of a Provider into a separate service.
+The [Daytona Server](/reference/server) communicates with Providers to execute operations relating to Workspace creation and lifecycle management.
+
+This makes it possible to create new Providers to govern Workspace management on any platform or hosting providers.
+
+## Managing Providers
+### Install a Provider
+You can install a Provider.
```

Comment:
I would remove these two lines. They seem redundant.

Reply:
For consistency with existing the docs, I'd rather keep this for now. We can revisit broad stylistic choices as part of a separate discussion; Niko and I are compiling a style guide to set out concrete rules for things like this. I mainly need a review on technical accuracy here.

This kind of intro (even if it seems redundant) is a common pattern in docs (e.g. [GitLab](https://docs.gitlab.com/ee/user/project/repository/#add-a-file-from-the-ui)).

Reply:
Okay, I'll leave you to that.